ethereum/contracts/interfaces/IERC4906.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.10 <0.9.0;
+
+import "@openzeppelin/contracts/interfaces/IERC165.sol";
+import "@openzeppelin/contracts/interfaces/IERC721.sol";
```

Comment:
Are these used?

Reply:
ah, no—they were part of the original OZ interface but should have removed